b1ddi/schema_dns_data_record_options_test.go

```diff
@@ -0,0 +1,80 @@
+package b1ddi
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func Test_DeepUpdateDataRecordOptions(t *testing.T) {
+	rq := require.New(t)
```

Comment:
Why separate require instance is needed?

Reply:
used assert instead